src/sentry/issues/issue_velocity.py

```diff
@@ -185,21 +223,37 @@ def get_latest_threshold(project: Project) -> float:
     ]
     client = get_redis_client()
     cache_results = client.mget(keys)  # returns None if key is nonexistent
-    threshold, stale_date = cache_results[0], cache_results[1]
-    now = convert_date_to_int(datetime.utcnow())
-    if (stale_date and int(stale_date) < now) or stale_date is None or threshold is None:
+    threshold = cache_results[0]
+    stale_date = None
+    if cache_results[1] is not None:
+        # TODO(isabella): update this once the old format is no longer being used
+        try:
+            stale_date = datetime.strptime(cache_results[1], STRING_TO_DATETIME)
+        except ValueError:
+            logger.info(
+                "issue_velocity.get_latest_threshold.old_date_format",
+                extra={"org_id": project.organization.id, "project_id": project.id},
```

Comment:
will use this to determine when the old date format is no longer in use (shouldn't take long to take those out)